soundmachine.service

```diff
@@ -7,7 +7,11 @@ Type=simple
 Restart=always
 RestartSec=2
 User=pi
-ExecStart=php /home/pi/align-state.php --quiet
+
+ExecStart=php /home/pi/soundmachine/align-state.php --quiet
```

Comment:
These paths are static and require someone using this utility to install the code in the exact same place.  I wonder how difficult it would be to create a script that would generate the appropriate service definition.  Then again, it might also be easier to just make the entire utility an executable.